config/application.yml.example

```diff
@@ -10,7 +10,7 @@
 
 development:
   allow_third_party_auth: 'yes'
-  domain_name: 'xxx.xxx'
+  domain_name: 'localhost:3000'
   ducksboard_uid: '8VEQre2c8b62mQV0sa'
   idp_sso_target_url: 'https://upaya-mockidp.18f.gov/saml/auth'
```

Comment:
or are we looking for something local/just a place holder since it's the `dev` env?